robocop/checkers/naming.py

```diff
@@ -662,3 +669,60 @@ def find_similar_variables(self, tokens, node):
                     col=token.col_offset,
                 )
             self.variables[normalized_token].add(token.value)
+
+
+class DeprecatedStatementChecker(VisitorChecker):
+    """Checker for finding deprecated statments."""
+
+    reports = (
+        "deprecated-statement",
+    )
+    # deprecated:alternative
+    deprecated_statements_rf3 = {}
+    deprecated_statements_rf4 = {"runkeywordunless": "IF/ELSE",
+                                 "runkeywordif":     "IF/ELSE"
+                                 }
+    deprecated_statements_rf5 = {"runkeywordunless": "IF/ELSE"}
```

Comment:
deprecated_statements_rf4 shall be empty and
deprecated_statements_rf5 shall have its contents but since tests were not running for rf5 (tests/atest/rules/naming/deprecated-statement/expected_output_rf5.txt ) I kept those to test

Reply:
Yeah since we operate on 5beta -> beta is still not yet 5 technically so it does not trigger. Leave # TODO fix before release comment so we edit this place when 5 will be out